UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Models/Response/BaseCallBackResponse.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Newtonsoft.Json;
+
+namespace UKHO.ExchangeSetService.Common.Models.Response
+{
+    public class BaseCallBackResponse
+    {
+        [JsonProperty("specversion")]
```

Comment:
Based on the formatting on BaseExchangeSetResponse.cs should this be specVersion?

Reply:
No, as per the exchange set service API spec this is correct format.
Reference ESS endpoint :-  /callback

Reply:
Thanks - Makes sense!